seed/static/seed/js/controllers/inventory_detail_controller.js

```diff
@@ -500,16 +500,32 @@ angular.module('BE.seed.controller.inventory_detail', [])
       };
 
       $scope.export_building_sync = function () {
```

Comment:
Not necessarily a code change for this spot, but...
Should we disable BS exports for non-BS imports? When there is not a "building_file" associated to a -State, the format isn't guaranteed to be right anyway... I think? 

Thinking about this more, maybe this doesn't need to be addressed now, but let me know what you think.

Reply:
It is able to export a file that should be valid, it's just likely to be veeery minimal. I'm sure if we should disable it or not

Reply:
Sorry - should have specified what I meant by "right" in my first comment. Would the export be a valid BuildingSync file or just valid xml if the -State didn't have a `building_file`?

Reply:
It should be a valid BuildingSync file, it's just going to have very little data probably